src/imitation/algorithms/adversarial/common.py

```diff
@@ -427,6 +430,45 @@ def _torchify_array(self, ndarray: Optional[np.ndarray]) -> Optional[th.Tensor]:
         if ndarray is not None:
             return th.as_tensor(ndarray, device=self.reward_train.device)
 
+    def _get_log_policy_act_prob(
+        self,
+        obs_th: th.Tensor,
+        acts_th: th.Tensor,
+    ) -> th.Tensor:
+        """Evaluates the given actions on the given observations.
+
+        Args:
+            obs_th: A batch of observations.
+            acts_th: A batch of actions.
+
+        Returns:
+            A batch of log policy action probabilities.
+        """
+        log_policy_act_prob, log_policy_act_prob_th = None, None
+        if isinstance(self.gen_algo.policy, policies.ActorCriticPolicy):
```

Comment:
```suggestion
        policy = self.gen_algo.policy
        if isinstance(policy, policies.ActorCriticPolicy):
```
You use policy a bunch here, so may as well shorten it.

Reply:
Good point